pkg/templates/nstemplatetiers/testtemplates/fakenstemplatetiers/test/cluster.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: test-cluster-resources
+objects:
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${SPACE_NAME}-deployments
+  spec:
+    quota:
+      hard:
+        count/deployments.apps: "30"
+        count/deploymentconfigs.apps: "30"
+        count/pods: "50"
+        count/virtualmachines.kubevirt.io: "2"
+    selector:
+      annotations: null
+      labels:
+        matchLabels:
+          toolchain.dev.openshift.com/space: ${SPACE_NAME}
```

Comment:
could you please point me to the place where these templates are currently used/defined? I cannot find anything corresponding to it.
My question is where (and why) did we get these files.